lstchain/reco/r0_to_dl1.py

```diff
@@ -54,6 +54,11 @@
 from ..calib.camera.r0 import LSTR0Corrections
 from ..pointing import PointingPosition
 
+logger = logging.getLogger('lstchain.reco.r0_to_dl1')
+logger.setLevel(logging.INFO)
```

Comment:
Logging setup like this should only happen in the main function of a script,  never as part of a library. 

Libraries only get the logger they need.

And using `__name__` is the right thing to do.

Reply:
done

Reply:
This is note fixed.

The three lines `setLevel`, `handler`, and `addHandler` should not be in this file but in the `main` function of the scripts.

Reply:
So in this script I keep `logger = logging.getLogger(__name__)` and the rest should be moved to the `main()` of `lstchain/scripts/lstchain_data_r0_to_dl1.py`?

Reply:
yes

Reply:
is now OK @MaxNoe?